dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/RumFeature.kt

```diff
@@ -68,7 +71,7 @@ internal object RumFeature : SdkFeature<Any, Configuration.Feature.RUM>() {
 
     internal var debugActivityLifecycleListener: Application.ActivityLifecycleCallbacks? = null
 
-    internal lateinit var vitalExecutorService: ScheduledThreadPoolExecutor
+    internal var vitalExecutorService: ScheduledExecutorService = NoOpScheduledExecutorService()
```

Comment:
do we introduce `NoOpScheduledExecutorService` because of `ProhibitLeavingStaticMocksIn`? If yes, then normally `lateinit` should be skipped by this check and no change is needed.

Reply:
Not because of that, it is just because is more readable this way otherwise I would have to make it an optional and will have then to check it everywhere. Forgot to remove the late init 👍